guides/common/modules/proc_enable-insights-with-capsule.adoc

```diff
@@ -0,0 +1,21 @@
+[id='register-capsule-with-insights_{context}']
+
+= Register a Capsule server with Red Hat Insights

```

Comment:
Please use the appropriate variable for Capsule server for the heading too. 
In Satellite docs and mod docs, and according to IBM style, we always use a gerund in our headings.